### DISCUSSION TITLE
iSeed v3.0 update is a breaking change release, here's what to do if you Laravel project is broken

### DISCUSSION THREAD

**Discussion by tihomiro** [orangehill/iseed#174] [General]
Hey there,

If you use a pre-v8 Laravel installation and you happen to have your composer.json set up so that it requires iseed package like this:

`"orangehill/iseed": "dev-master",`

make sure to change it to:

`"orangehill/iseed": "~2.6.4",`

otherwise, your code might break due to a different seed directory folder name. Hope this solves someone's problem!

**Comment by SEJohnsen** [comment]
iseed 3.0 is not compatible with Laravel 5.5-5.8. It's not just the folder location but the syntax as well. Using ~2.6.4 helped a lot!
